front/src/Messages/MakeUnsafe.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import type {UnsafeString} from "./UnsafeString";
+
+type ReplaceReturnType<T extends (...a: any) => string> = (...a: Parameters<T>) => UnsafeString;
```

Comment:
this may get a bit unreadable but you probably also want to replace return types that return an object with a string property